static/app/components/replays/replayContext.tsx

```diff
@@ -479,37 +480,64 @@ function ProviderNonMemo({
         return null;
       }
 
+      const hasGestures = events?.some(e => e.type === 3);
```

Comment:
What do you think about having this logic inside of `VideoReplayerWithInteractions` instead?

Reply:
i thought about that but it seems worse because we'd have to conditionally instantiate the rrweb `Replayer`, and then in all the `play()`, `pause()` etc functions we'd also have to conditionally call the `Replayer` functions depending on if the rrweb inst exists - feels like way more things to change